arrow-libs/optics/arrow-optics/src/commonTest/kotlin/arrow/optics/test/laws/Laws.kt

```diff
@@ -9,8 +11,9 @@ interface LawSet {
   val laws: List<Law>
 }
 
-data class Law(val name: String, val test: suspend TestScope.() -> Unit)
+data class Law(val name: String, val test: suspend TestScope.() -> Any?)
 
+@IgnorableReturnValue
 fun <A> A.equalUnderTheLaw(b: A, f: (A, A) -> Boolean = { x, y -> x == y }): Boolean =
```

Comment:
Since this is test code, I'd rather just change this to return `Unit` since it can never return `false`